lib/model/model.go

```diff
@@ -2066,6 +2064,14 @@ func (m *Model) internalScanFolderSubdirs(ctx context.Context, folder string, su
 						Deleted:    true,
 						Version:    f.Version.Update(m.shortID),
 					}
+					// We do not want to override the global version
```

Comment:
Move the comment inside the if, I was super confused about why we would not want to override the global file until I read on and realized it was for a corner case.

It's also confusing that we are inside an `!f.IsInvalid()` case and yet it may be invalid, but I guess that's Ok...

Reply:
I dropped the if entirely from this PR. That is the change of behaviour introduced in #4750 and thus should be done there (didn't realize it when splitting this PR from the other one).